File: translation_output/codellama/codenet/python/java/p03250.java
<s> [INST] <<SYS>>
You are an expert java programmer and assistant
<</SYS>>

python code:

abc = list(map(int, input().split()))
abc.sort()
print(int(str(abc[2]) + str(abc[1])) + abc[0])

Translate the above python code to java.

java code:

[/INST]

Here is the Java code that translates the Python code:
```
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] numbers = input.split(" ");
        int[] intNumbers = new int[numbers.length];
        for (int i = 0; i < numbers.length; i++) {
            intNumbers[i] = Integer.parseInt(numbers[i]);
        }
        Arrays.sort(intNumbers);
        int sum = intNumbers[2] + intNumbers[1] + intNumbers[0];
        System.out.println(sum);
    }
}
```
This code first reads the input from the user using the `Scanner` class. It then splits the input into an array of strings using the `split` method. It then converts each string to an integer using the `Integer.parseInt` method and stores it in an array of integers.

Next, it sorts the array of integers using the `Arrays.sort` method. Finally, it calculates the sum of the first three elements of the sorted array and prints it to the console using the `System.out.println` method.

Note that the Python code uses the `map` function to convert the input to an array of integers, but in Java, we can use the `Integer.parseInt` method to convert each string to an integer.</s>

File: translation_output/codellama/codenet/java/python/p03250.py
<s> [INST] <<SYS>>
You are an expert python programmer and assistant
<</SYS>>

java code:

import java.util.Scanner;

public class p03250 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		String StringA = String.valueOf(a);
		String StringB = String.valueOf(b);
		String StringC = String.valueOf(c);
		
		if (a >= b && a >= c) {
			if (b  >= c) {
				System.out.println(Integer.parseInt(StringA + StringB)+ c);
			} else  {
				System.out.println(Integer.parseInt(StringA + StringC)+ b);
			}
		} else if (b>=a && b >= c){
			if (a>=c) {
				System.out.println(Integer.parseInt(StringB + StringA) + c);
			} else {
				System.out.println(Integer.parseInt(StringB + StringC)+a);	
			}
		} else {
			if(a>= b) {
				System.out.println(Integer.parseInt(StringC + StringA)+b);
			} else {
				System.out.println(Integer.parseInt(StringC + StringB)+a);
			}
		}

	}
}


Translate the above java code to python.

python code:

[/INST]
```python
import sys

a, b, c = map(int, sys.stdin.readline().split())

if a >= b and a >= c:
    if b >= c:
        print(a + b + c)
    else:
        print(a + c + b)
elif b >= a and b >= c:
    if a >= c:
        print(b + a + c)
    else:
        print(b + c + a)
else:
    if a >= b:
        print(c + a + b)
    else:
        print(c + b + a)
```
Note: The `map` function is used to convert the input from strings to integers. The `sys.stdin.readline()` function is used to read a line of input from the user. The `split()` function is used to split the input into a list of strings. The `int()` function is used to convert each string to an integer.</s>
